### PR TITLE
Improved documentation to DiseasyObservables

### LINKED ISSUE #42: DiseasyObservables: Add vignette that describes basic principles
The `DiseasyObservables` is a central module that should have its own vignette describing the basic principles.

See our [documentation standard](https://github.com/ssi-dk/diseasy/wiki/Diseasy-documentation-standard#super-documentation) for details

### LINKED ISSUE #46: DiseasyBaseModule: add option to control logging
Presently, the logger is always turned on for the modules.
To give the user better control of when logging should occur, options should be added to control the logging.

### PR DESCRIPTION
**Pull request by RasmusSkytte**
### Intent
This PR does a number of things
1) it improves the roxygen documentation for `DiseasyObservables` (Fixes parts of #37)
2) It adds a vignette that describes its basic principles (Fixes #42)
3) It adds a toggle to `DiseasyBaseModule` that gives control of logging (Fixes #46)

### Approach
1) N/A
2) N/A
3) `DiseasyBaseModule` gets a `logging` argument and `options("diseasy.logging" = FALSE)`

### Known issues
* [x] PR #53 needs to be merged before test will work (there is a issue with `printr` that this PR fixes)
* [x] PR https://github.com/ssi-dk/diseasystore/pull/64 needs to be merged before tests will work

### Checklist

* [x] The PR passes all local unit tests
* [ ] I have documented any new features introduced
* [ ] If the PR adds a new feature, please add an entry in `NEWS.md`
* [ ] A reviewer is assigned to this PR

### CODE REVIEW

File: vignettes/diseasy-observables.Rmd [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+obs$set_case_definition(case_definition = "Google COVID-19")
+```
+
+To see the data that comes with underlying `diseasystore` we can query the module.
+```{r available_observables}
+obs$available_observables
+```
+
+```{r available_aggregations}
+setdiff(obs$ds$available_features, obs$available_observables)
```
Comment (RasmusSkytte) [inline vignettes/diseasy-observables.Rmd:62]:
```suggestion
obs$available_aggregations
```

Comment (RasmusSkytte) [reply vignettes/diseasy-observables.Rmd:62]: When #39 is merged, we get access to `obs$available_aggregations` instead

Comment (RasmusSkytte) [reply vignettes/diseasy-observables.Rmd]: PR #39 added directly to this PR and this suggestion is now implemented.

File: vignettes/diseasy-observables.Rmd [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+```{r initialization}
+obs <- DiseasyObservables$new(
+  conn = DBI::dbConnect(RSQLite::SQLite())
+)
+# NOTE: Alternatively we could set options("diseasy.conn" = ...)
+
+obs$set_case_definition(case_definition = "Google COVID-19")
+```
+
+To see the data that comes with underlying `diseasystore` we can query the module.
```
Comment (telkamp7) [inline vignettes/diseasy-observables.Rmd:56]: Typo: 'that comes with _the_ underlying `diseasystore` ...'

File: vignettes/diseasy-observables.Rmd [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+obs$available_aggregations
+```
+
+To check the current status of the module, the `$describe()` method can be used:
+```{r describe}
+obs$describe()
+```
+
+# Getting observations
+We can query the model to gives the data for a given observable in a given time frame:
```
Comment (telkamp7) [inline vignettes/diseasy-observables.Rmd:71]: Typo: 'the model to _give_ the data for a ...'

**Review by telkamp7 — APPROVED** [review]

File: vignettes/diseasy-observables.Rmd (hunk 2 above)
Comment (RasmusSkytte) [reply vignettes/diseasy-observables.Rmd:56]: Thanks!

**Pull request merged**